metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/configuration/metron-enrichment-env.xml

```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<configuration supports_final="true">
+  <property>
+    <name>geoip_url</name>
+    <value>http://geolite.maxmind.com/download/geoip/database/GeoLite2-City.mmdb.gz</value>
+    <description>Location of the GeoIP data to load.</description>
+    <display-name>GEOIP Load Datafile URL</display-name>
+  </property>
+  <property>
+    <name>enrichment_host_known_hosts</name>
+    <description>List of Known Hosts for Host Enrichment</description>
+    <value>[{"ip":"10.1.128.236", "local":"YES", "type":"webserver", "asset_value" : "important"},{"ip":"10.1.128.237", "local":"UNKNOWN", "type":"unknown", "asset_value" : "important"},{"ip":"10.60.10.254", "local":"YES", "type":"printer", "asset_value" : "important"}]</value>
+    <display-name>Host Enrichment</display-name>
+    <value-attributes>
+      <type>content</type>
+    </value-attributes>
+  </property>
+  <property>
+    <name>enrichment_kafka_start</name>
+    <description>Enrichment Topology Spout Offset</description>
+    <value>UNCOMMITTED_EARLIEST</value>
+    <display-name>Enrichment Offset</display-name>
```

Comment:
With Ambari's drop down selector, all I can read is "UNCOMMITTE".  I can't actually tell if it is UNCOMITTED_EARLIEST or UNCOMMITTED_LATEST.  Is that a pre-existing condition?